### PR TITLE
Update cloud_setup role

### PR DESCRIPTION
**Pull request by afgane**

### CODE REVIEW

File: extra-files/cloud_setup/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The files described below are the ones the user can modify in order to meet thei
 
 * `vimrc` : The VIM configurations used for the remote target.
 
-* `usegalaxy_main_tool_list.yml` : This is a replica of the Galaxy main tool list for the most part, but edited to fit the size of the instance. The Galaxy main tool list can be obtained using this ephemeris [script](https://github.com/galaxyproject/ephemeris/blob/master/ephemeris/get_tool_list_from_galaxy.py). This file can be changed in group_vars/cloud_setup if the user wishes to use a custom tool list. While making a new tool list, it is important to keep in mind the tags in the `yaml` file.
+* `usegalaxy_main_tool_list.yml` : This is a replica of the Galaxy main tool list for the most part, but edited to fit the size of the instance. The Galaxy Main tool list can be obtained using this ephemeris [script](https://github.com/galaxyproject/ephemeris/blob/master/ephemeris/get_tool_list_from_galaxy.py). This file can be changed in group_vars/cloud_setup if the user wishes to use a custom tool list. While making a new tool list, it is important to keep in mind the tags in the `yaml` file.
```
Comment (drosofff) [inline extra-files/cloud_setup/README.md]: What does "keep in mind" mean ?

Comment (drosofff) [inline extra-files/cloud_setup/README.md]: What does "keep in mind" mean ?

File: extra-files/cloud_setup/README.md [hunk 2]
```diff
@@ -39,12 +39,10 @@ This tool uses `tool_panel_section_label` as we are making a new section in the
...
- * `dependency_resolvers_conf.xml` : Modified `dependency_resolvers_conf` file with conda moved up as the first option.
-
  * `data_manager_tool_list.yml` : List of data managers installed.
 
  * `data_manager_tasks.yml` : Run data manager tasks with dbkeys (organisms).
 
  * `fix_hostname.sh` : Custom script to fix the hostname on Jetstream instances.
 
- *  `tool_sheds_conf.xml` : Two tool sheds are available right now through this file, if the user has a local toolshed, they can add it to this list.
+ * `tool_sheds_conf.xml` : Two tool sheds are available right now through this file, if the user has a local ToolShed, they can add it to this list.
```
Comment (drosofff) [inline extra-files/cloud_setup/README.md]: Two tool sheds are available right now through this file (Galaxy main and test toolsheds). Users can reference other toolsheds in the file.

File: roles/cloud_setup/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+This step is required because `galaxy-tools` role flushes all the handlers so
+if not split up, the cleanup step (_Run #3_) will run before any of the tools
+are installed.
+ * In `group_vars/cloud_setup`, set `galaxy_tools_delete_bootstrap_user: yes`
+ * To speed up the build, in `galaxy.yml`, comment out all the roles except
+   `galaxyprojectdotorg.galaxy-tools`
+
+### Run #3: Cleanup the image for bundling
+ * Revert changes from _Run #2_ and uncomment tasks from `cleanup.py`
+ * In `galaxy.yml`, comment out all the roles except `cloud_setup`
```
Comment (drosofff) [inline roles/cloud_setup/README.md]: Just a question: would it be possible to link the three steps using an alternate galaxy_jetstream.yml ?

File: extra-files/cloud_setup/README.md (hunk 1 above)
Comment (afgane) [inline extra-files/cloud_setup/README.md]: It's essentially the same as 'remember'.

Comment (drosofff) [inline extra-files/cloud_setup/README.md]: Yes this I got it :-) but remember what for what (in the process of making a new tool list) is less clear for me

Comment (afgane) [inline extra-files/cloud_setup/README.md]: Sorry! It wasn’t clear what the intent was the way it was posted :)
I'll update.

Comment (afgane) [inline extra-files/cloud_setup/README.md]: Updated now.

**Comment by drosofff** [comment]
Poke @mvdbeek

**Comment by mvdbeek** [comment]
Failure is unrelated ansible bug https://github.com/ansible/ansible/issues/17920.
Fortunately 2.1.2.0 is out, and it's older than the referenced commit, so we can finally switch to a stable version.

**Comment by mvdbeek** [comment]
opening/closing to get travis-ci to pick up the test that seemed to be in limbo

**Comment by mvdbeek** [comment]
awesome, thanks @afgane and @drosofff!

**Pull request merged**